public/scripts/projects.js

```diff
@@ -1,30 +1,57 @@
+// 1. break out the js files to CMV files.
+// 2. refractor code to make it more eefficient and lean
+
 
 'use strict';
 var projects = [];
 
+//Models
 function Project(rawDataObject) {
   for (var key in rawDataObject) {
     this[key] = rawDataObject[key];
   }
 }
+
+//refractored - broken
+// Project.prototype.toHtml = () => {
+//   let template = $('#project-template').html();
+//   let templateRender = Handlebars.compile(template);
+//   return templateRender(this);
+// };
+
+// OLD WAY
 Project.prototype.toHtml = function() {
   var template = $('#project-template').html();
   var templateRender = Handlebars.compile(template);
   return templateRender(this);
 };
 
-projects.forEach(function(NewProjectObject) {
-  $('#projects').append(NewProjectObject.toHtml());
-});
+//Controller
+//refractored -works!!
+projects.forEach = (NewProjectObject) => $('#projects').append(NewProjectObject.toHtml);
+
+// OLD WAY
+//projects.forEach(function(NewProjectObject) {
+//   $('#projects').append(NewProjectObject.toHtml());
+// });
+
 
 Project.all = [];
+
+//Models
+//Refractor - broken. Can I have 2 arrow functions?
```

Comment:
yes you can but your syntax here is wrong. you have rawData.forEach(ele) you need forEach(ele => function contents)